enterprise/internal/insights/background/queryrunner/worker.go

```diff
@@ -131,13 +131,13 @@ func getRateLimit(defaultValue rate.Limit) func() rate.Limit {
 
 // NewResetter returns a resetter that will reset pending query runner jobs if they take too long
 // to complete.
-func NewResetter(ctx context.Context, workerStore dbworkerstore.Store, metrics dbworker.ResetterMetrics) *dbworker.Resetter {
+func NewResetter(ctx context.Context, logger log.Logger, workerStore dbworkerstore.Store, metrics dbworker.ResetterMetrics) *dbworker.Resetter {
 	options := dbworker.ResetterOptions{
 		Name:     "insights_query_runner_worker_resetter",
 		Interval: 1 * time.Minute,
 		Metrics:  metrics,
 	}
-	return dbworker.NewResetter(workerStore, options)
+	return dbworker.NewResetter(logger.Scoped("dbworker.Resetter", ""), workerStore, options)
```

Comment:
```suggestion
	return dbworker.NewResetter(logger, workerStore, options)
```
The logger here is already scoped to `queryrunner.Resetter`